node-definitions/freebsd/FreeBSD.yaml

```diff
@@ -0,0 +1,66 @@
+id: FreeBSD
+general:
+  nature: server
+  description: FreeBSD with "nuage-init" (a limited version of cloud-init).
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - em0
+    serial_ports: 1
+    loopback:
+      - lo0
+ui:
+  visible: true
+  label_prefix: fBSD-
+  icon: server
+  label: FreeBSD
+sim:
+  linux_native:
+    libvirt_domain_driver: kvm
+    driver: server
+    disk_driver: sata
+    ram: 1024
+    cpus: 1
+    cpu_limit: 100
+    nic_driver: e1000
```

Comment:
FreeBSD can make use of VirtIO in the default kernel.  I think it performs a bit better.